src/test/java/org/usrv/http/ServerTests.java

```diff
@@ -171,6 +171,31 @@ void serverRespondsWith400() throws Exception {
         }
     }
 
+    @Test
+    @DisplayName("Server responds with a 400 status if the request is missing host for HTTP 1.1")
+    void serverRespondsWith400WhenMissingHost() throws Exception {
+        try (Socket socket = new Socket("localhost", 80);
+             PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
+             BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()))) {
+
+            // Send a malformed request
```

Comment:
Either remove this comment, or edit it to say something with "...missing a host header"